internal/io/http/rest_sink_test.go

```diff
@@ -565,3 +597,9 @@ func TestRestSinkMethod(t *testing.T) {
 	err := rs.Validate(map[string]interface{}{"method": "head"})
 	require.Error(t, err)
 }
+
+func TestRestSinkConfigure(t *testing.T) {
```

Comment:
Better to make this test name clearly showing it's a failure test. I guess the intention is to test unsupported compression method.

Reply:
fixed